src/destiny.cpp

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Imo this is a good start. Just a little nit-pick:

Can you rename it to ``Game_Destiny`` similiar to how Ineluki patch is implemented (which is ``Game_Ineluki``).

Then make it a class and put it the variable in ``main_data.h/cpp`` and instantiate it in ``ResetGameObjects``.

Also please update ``Makefile.am`` and ``CMakeLists.txt`` by adding the new files to it (put them at the correct location in alphabetical order).

-----------

Yes I know that DynRPG does it in a different way but I plan to rewrite this as less global state makes the cleanup easier when switching the game.